repl/variables.go

```diff
@@ -0,0 +1,38 @@
+package repl
+
+import (
+	"os"
+	"strings"
+
+	"github.com/spacemeshos/go-spacemesh/log"
+)
+
+func setVariables(values map[string]string) error {
+	for key, value := range values {
+		err := os.Setenv(key, value)
```

Comment:
By environment variables we meant variables contained inside the node running and not the OS level. This might be problematic if one runs more than one node on a machine and the variables clash